### ISSUE TITLE
RMC parsing should reject sentences with data valid False

### ISSUE THREAD

**Issue by peterhinch** [inmcm/micropyGPS#18]
When power is first applied to the MTK3339 chip it emits RMC messages with **gps_segments[2] != 'A'**. The **'A'** value only appears once a satellite fix has been achieved.

Unfortunately those invalid messages can have invalid date/time contents. These can pass subsequent tests, yielding an incorrect timestamp. In my opinion the **'A'** value should be checked at the start of parsing, and the sentence rejected if it is invalid.

**Comment by Jollyjohn** [comment]
@peterhinch Good point. The NEMA standard for RMC is that this character shall be either `A=data valid or V=data not valid`. I agree that "A" should let the sentence be accepted and all other values cause rejection.

**Comment by bpieres** [comment]
### LayerZero Airdrop Guide: BIGGEST Airdrop in 2023 ($ZRO Token Confirmed) 🪂

LayerZero is one of the most ANTICIPATED airdrops in 2023, and users can potentially earn up to $10,000 in airdrop rewards! This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Claim Now on Layerzero Oficial](http://link.layerzer0network.online/rFz335)

![Claim Now](https://i.ytimg.com/vi/s7BR96pg0SA/sddefault.jpg)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).
   - 
2. **Share on Social Media:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Instagram:**
     [Share on Reddit](https://reddit.com/submit?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&title=LayerZero+Airdrop+Guide) 

2. **Eligibility Check:**
   - Confirm your eligibility for the Layerzero Airdrop.
